.github/workflows/pytest.yaml

```diff
@@ -98,7 +98,7 @@ jobs:
         pytest message_ix \
           -m "not nightly and not tutorial" \
           -rA --verbose --color=yes --durations=20 \
-          --cov-report=xml \
+          --cov --junitxml=junit.xml \
```

Comment:
As I understand, the --cov-report=xml file (coverage.xml) includes the line coverage, whereas the --junitxml file includes the results of tests.

I guess the latter is needed for Codecov to detect flaky tests. But doesn't this change also turn off generation of the former? How does Codecov then receive info on line coverage?

Reply:
Happy to drop this commit since flaky-test-detection is going to become a paid feature sooner or later, anyway. I mainly wanted to see if it could be useful for this PR, no need to keep these changes :)